src/AutoRest.CSharp/Common/Output/Models/Types/ObjectTypeProperty.cs

```diff
@@ -221,10 +223,5 @@ private static FormattableString CreateExtraPropertyDiscriminatorSummary(CSharpT
             }
             return updatedDescription ?? $"";
         }
-
-        public override string ToString()
-        {
-            return $"ObjectTypeProperty {{Name: {Declaration.Name}, Type: {Declaration.Type}}}";
-        }
```

Comment:
this method was introduced by me a long time ago to let the debugger show more information about this type. We never need to call this in our code.
Therefore in this PR, I replaced this method with `DebuggerDisplay`